docs/tutorials/amd-igpu-780m.md

```diff
@@ -0,0 +1,97 @@
+
```

Comment:
Yes. I will updated with a title. Thank you.

Reply:
We might want a more generic title (e.g. leaving off the 780m) since this document could be expanded to include other AMD iGPUs. The new Ryzen 9 HX 375 will probably get good performance so will probably be added, but there'd be little reasons to add an additional file when it could be appended to this.